openzeppelin-test-helpers.js

```diff
@@ -4,6 +4,7 @@ module.exports = {
   balance: require('./src/balance'),
   BN,
   constants: require('./src/constants'),
+  deploy: require('./src/deploy'),
```

Comment:
I'm not convinced that this is a good name. It's too generic when the purpose of this module is very specific. In other words, people might think this would allow them to deploy an arbitrary contract, but it will be specifically for this kind of singleton global contracts. Are there any alternative module names you've considered?

Reply:
Okay after reading the rest of the PR it seems quite trivial what these are. Not sure if comments would help much.

Reply:
The _current_ purpose of the module is very specific: I left it open-ended to allow for other, future utilities here.

Reply:
But do you agree with my point that a user will expect the `deploy` module to somehow help them deploy their own contracts?

I'm okay with a bit of open-endedness, but I'd name the module something like `singletons`.